### ISSUE TITLE
Error BC30149	while compiling using .NET Core Preview 5/6

### ISSUE THREAD

**Issue by rrvenki** [Nukepayload2/NetCore3PreVBWinExe#1]
A code generated using C# template (Wpf Core) converted into VB in .NET Core Preview 5/6 gives below error.

Error	BC30149	Class 'App' must implement 'Sub InitializeComponent()' for interface 'IComponentConnector'.

This error appears in App.g.vb which is autogenerated file. Change line 55 in App.g.vb to

`Public Sub InitializeComponent() Implements system.Windows.Markup.IComponentConnector.InitializeComponent`

Compiler will clear all errors.  But however when we run the app, App.g.vb will be regenerated and `Implements system.Windows.Markup.IComponentConnector.InitializeComponent` will be again missing in newly generated App.g.vb file.

More details are discussed [here](https://github.com/dotnet/core/issues/2080). Any ideas to interoperate "App" and "Application" while CS/VB conversions?

**Comment by rrvenki** [comment]
Found a solution from .NET WPF (not Core) from [here](https://stackoverflow.com/questions/35827046/vb-net-getting-error-class-application-must-implement-sub-initializecomponen)
1. Remove StartupURI line in App.xaml file. This should be in "Application" tag.
2. Change App.xaml.vb content as below.

```
Imports System.Windows
Partial Public Class App
    Public Shared Sub Main()
        Dim app As Application = New Application()
        app.Run(New MainWindow())
    End Sub
End Class
```

3. Goto project properties, Application tab, and select "Startup object" as "Sub Main".
Now the project should compile and work. I found a working solution after posting the issue. Hence closing it.
